engine/session.js

```diff
@@ -227,6 +228,17 @@ class Session {
           if (tickInterval <= 0.5) {
             tickInterval = 0.5;
           } else if (tickInterval > (this.maxTickInterval / 1000)) {
+            const change = ceil(abs(tickInterval - this.maxTickInterval));
```

Comment:
I saw that tickInterval is in seconds.
Needs indentation clean up.
Setting tick interval to the max limit doesn't necessarily need the limit to be changed. 
I have seen are cases where the interval goes over max once, then corrects itself at the next increment.
Also, regarding the warning message. Consider the case when you have multiple channels running.
May I suggest? something like:

```js
            const diffMs = ceil(abs((tickInterval * 1000) - this.maxTickInterval));
            if (!config.maxTickInterval) {
              if (numberOfLargeTicks > 2) {
                this.maxTickInterval += diffMs;
                numberOfLargeTicks = 0;
              } else {
                numberOfLargeTicks++;
              }
            } else {
              console.warn(`[${this._sessionId}]: Playhead tick interval went over Max tick interval by ${diffMs}ms.
              If the value keeps increasing, consider increasing the 'maxTickInterval' in engineOptions`);
            }   
```

Reply:
@Sgtfishtank Hey?